development/compiling/compiling_for_windows.rst

```diff
@@ -24,6 +24,12 @@ For compiling under Windows, the following is required:
   **Important:** When using MinGW to compile the ``master`` branch, you need GCC 9 or later.
 - `Python 3.5+ <https://www.python.org/downloads/windows/>`_.
   **Make sure to enable the option to add Python to the ``PATH`` in the installer.**
+
+  .. note:: There is a Microsoft Store package of Python but it is mainly for interactive use and not full development.
+           Because of restrictions on Microsoft Store apps, Python scripts may not have full write access to shared locations such as TEMP and the registry.
```

Comment:
```suggestion
  .. note::
  
           There is a Microsoft Store package of Python but it is mainly for interactive use and not full development.
           Because of restrictions on Microsoft Store apps, Python scripts may not have full write access to shared locations such as TEMP and the registry.
```

Though this can work, an empty line is required by convention.